src/main/resources/templates/job.xml

```diff
@@ -15,6 +15,14 @@
                 </parameterDefinitions>
             </hudson.model.ParametersDefinitionProperty>
         {% endif %}
+        <jenkins.model.BuildDiscarderProperty>
+            <strategy class="hudson.tasks.LogRotator">
+                <daysToKeep>{{ STORE_BUILDS_DAYS }}</daysToKeep>
+                <numToKeep>{{ STORE_BUILDS_TOTAL }}</numToKeep>
+                <artifactDaysToKeep>{{ STORE_ARTIFACTS_DAYS }}</artifactDaysToKeep>
+                <artifactNumToKeep>{{ STORE_ARTIFACTS_TOTAL }}</artifactNumToKeep>
```

Comment:
If they are not set and then we save through the UI they are set in the template as -1. I can default to this value maybe if not set?